cves/kernel/CVE-2021-27363.yml

```diff
@@ -194,8 +195,8 @@ specification:
 
     The answer field should be boolean. In answer_note, please explain
     why you come to that conclusion.
-  note:
-  answer:
+  note: "did not properly restrict access to transport handle attributes"
```

Comment:
What specification does this violate?

Reply:
There was a specification mentioned in one of the emails that it said it violated. I'll look into that again and update my answer based on that.